### PR TITLE
Bigquery insert query defaults

### PR DESCRIPTION
**Pull request by oravi**

### CODE REVIEW

**Comment by oravi** [comment]
Fixes #152

File: macros/edr/system/system_utils/get_config_var.sql [hunk 1]
```diff
@@ -30,12 +38,17 @@
...
     'query_max_size': 1000000,
     'insert_rows_method': 'max_query_size'
   } %}
+  {{- return(default_config) -}}
+{%- endmacro -%}
 
-  {{ return(var(var_name, default_config.get(var_name))) }}
-{% endmacro %}
+{%- macro bigquery__get_default_config() -%}
+    {% set default_config = elementary.default__get_default_config() %}
```
Comment (elongl) [inline macros/edr/system/system_utils/get_config_var.sql:51]: Not that important as we can't have the same dbt process run on two different adapters,
but you're overriding the default config for all adapters.

```suggestion
    {% set default_config = elementary.default__get_default_config().copy() %}
```

If we'd have logic that compares different config values among adapters, this would break it.

File: macros/edr/system/system_utils/get_config_var.sql [hunk 2]
```diff
@@ -20,6 +27,7 @@
     'disable_skipped_model_alerts': true,
     'disable_skipped_test_alerts': true,
     'dbt_artifacts_chunk_size': 5000,
+    'test_sample_row_count': 5,
```
Comment (elongl) [inline macros/edr/system/system_utils/get_config_var.sql:30]: Those are unrelated changes, any reason they're on this PR?

File: macros/edr/system/system_utils/get_config_var.sql [hunk 3]
```diff
@@ -30,12 +38,17 @@
       'column_numeric': ['min', 'max', 'zero_count', 'zero_percent', 'average', 'standard_deviation', 'variance']
     },
     'time_format': '%Y-%m-%d %H:%M:%S',
-    'long_string_size': 16384,
+    'long_string_size': 65535,
```
Comment (elongl) [inline macros/edr/system/system_utils/get_config_var.sql:41]: Same as above.

File: macros/edr/system/system_utils/get_config_var.sql (hunk 1 above)
Comment (oravi) [reply macros/edr/system/system_utils/get_config_var.sql:51]: I don't think that it's important, also don't want to pay the copy for every var access
